compiler/control/OMROptions.hpp

```diff
@@ -1523,6 +1523,8 @@ class OMR_EXTENSIBLE Options {
         _inlinerCGColdBorderFrequency = 0;
         _inlinerCGVeryColdBorderFrequency = 0;
         _alwaysWorthInliningThreshold = 0;
+        _serverInlinerBorderFrequency = 0;
```

Comment:
These should be initialized with -1 based on the code in OpenJ9.

Reply:
It is, in `OMR::Options::jitPreProcess`; I'm keeping it consistent with `_inlinerBorderFrequency`.